test-requirements.txt

```diff
@@ -2,7 +2,7 @@ coverage==3.7.1
 ddt==0.8.0
 nose==1.3.4
 pep8==1.5.7
-edx-lint==0.4.3
+edx-lint==0.5.5
```

Comment:
We should switch this over to pip-tools later.

Reply:
^ actually, we might be able to rip that out altogether. There are some python scripts in here that I am not convinced that the mobile team still needs. If that is the case, we can remove all things python from this repo (but in a separate PR)